README.md

```diff
@@ -1,11 +1,272 @@
+    linkStyle 20 stroke:#2962FF,fill:none
+    linkStyle 21 stroke:#AA00FF,fill:none
+```
+Key: * = coming soon
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Frontend</summary>
+  <ul>
+    <li>Framework: <a href="https://react.dev/">React</a></li>
+    <li>Language: <a href="https://www.typescriptlang.org/">Typescript</a></li>
+    <li>Testing Library: 
+      <ul>
+        <li><a href="https://jestjs.io/">Jest</a></li>
+        <li><a href="https://testing-library.com/">Testing Library</a></li>
+      </ul>
+    </li>
+    <li>UI Library: <a href="https://ant.design/">Ant Design</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Backend</summary>
+  <ul>
+    <li>Framework: <a href="https://....">Node.js</a> with <a href="https://expressjs.com/">Express</a></li>
+    <li>Language: <a href="https://www.typescriptlang.org/">Typescript</a></li>
+    <li>Testing Library:<a href="https://jestjs.io/">Vitest</a></li>
+    <li>API Documentation: 
+      <ul>
+        <li><a href="https://swagger.io/">Swagger</a></li>
+        <li><a href="https://learn.openapis.org/">OpenAPI</a></li>
+      </ul>
+    </li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features
+- **[Sign in with Frequency](https://github.com/AmplicaLabs/siwf)**
+- **Create a post**
+- **Create a comment on a post**
+- **View a feed**
+- **View other users' profiles**
+- **Follow a user**
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- LIVE Docs -->
+
+## 🚀 Live Docs
+
+- [Gateway Live Docs](https://amplicalabs.github.io/gateway/)
+- Open Api docs coming soon...
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+
+- ### Frontend
+
+  See [Frontend Readme](./frontend/README.md)
+
+- ### Backend
+
+  See [Backend Readme](./backend/README.md)
+
+### Prerequisites
+
+In order to run this project you need to become a provider. To do so, visit our [Provider Dashboard](https://provider.frequency.xyz/)!
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  git clone git@github.com:AmplicaLabs/social-app-template.git
+  cd social-app-template
+```
+
+### Deployment
+
+You can deploy using our containers. Check the [docker-compose.yaml](backend/docker-compose.yaml) file for more details.
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+- [Contributing Guidelines](./CONTRIBUTING.md)
+- [Open Issues](https://github.com/AmplicaLabs/social-app-template/issues)
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support
+
+
+If you like this project...join the Zoom calls or loom at the label for `good-first-issue`
```

Comment:
I though we can drop the support section for this repo?

We don't really have zoom calls or such for gateway.